cmd/util/opts/storage.go

```diff
@@ -70,7 +70,9 @@ func (o *StorageOpt) Set(value string) error {
 			return fmt.Errorf("unpexted key %s in field %s", key, field)
 		}
 	}
-	storageSpec, err := legacy_job.ParseStorageString(sourceURI, destination, options)
+	// TODO(forrest) [correctness]: need to allow aliases to be provided over CLI
+	alias := "TODO"
```

Comment:
Need to address, not sure if the alias should be passed by the client or picked by the CLI.